generated/nidcpower/nidcpower/enums.py

```diff
@@ -512,14 +512,18 @@ class OutputFunction(Enum):
 
 
 class OutputStates(Enum):
-    VOLTAGE = 0
+    CONSTANT_VOLTAGE = 0
     r'''
     The channel maintains a constant voltage by adjusting the current.
     '''
-    CURRENT = 1
+    CONSTANT_CURRENT = 1
     r'''
     The channel maintains a constant current by adjusting the voltage.
     '''
+    INHIBITED = 1163
```

Comment:
@marcoskirsch Any objections to the use of `INHIBITED` vs `OUTPUT_INHIBITED`?
Seems like a reasonable name to me.

Reply:
What is it called in C, LabVIEW, C#?